src/style.css

```diff
@@ -0,0 +1,47 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, auto));
+}
+
+@media all and (min-width: 1200px) {
```

Comment:
not necessary

Reply:
without media cells will shrink to 300px at 1200px screen